k8s/pkg/apis/longhorn/v1beta2/replica.go

```diff
@@ -10,7 +10,10 @@ const (
 )
 
 const (
-	ReplicaConditionTypeRebuildFailed = "RebuildFailed"
+	ReplicaConditionTypeRebuildFailed                = "RebuildFailed"
+	ReplicaConditionTypeWaitForBackingImage          = "WaitForBackingImage"
+	ReplicaConditionReasonWaitForBackingImageFailed  = "GetBackingImageFailed"
+	ReplicaConditionReasonWaitForBackingImageWaiting = "Waiting"
 
```

Comment:
Whats the difference between `ReplicaConditionReasonWaitForBackingImageWaiting` and `ReplicaConditionTypeWaitForBackingImage`

Reply:
One is `Reason`, the other one is `Type`
A condition is made of 
```
type Condition struct {
	// Type is the type of the condition.
	Type string `json:"type"`
	// Status is the status of the condition.
	// Can be True, False, Unknown.
	Status ConditionStatus `json:"status"`
	// Last time we probed the condition.
	LastProbeTime string `json:"lastProbeTime"`
	// Last time the condition transitioned from one status to another.
	LastTransitionTime string `json:"lastTransitionTime"`
	// Unique, one-word, CamelCase reason for the condition's last transition.
	Reason string `json:"reason"`
	// Human-readable message indicating details about last transition.
	Message string `json:"message"`
}
```